src/Nethermind/Nethermind.Network.Discovery/NettyDiscoveryHandler.cs

```diff
@@ -67,26 +67,24 @@ public override void ChannelReadComplete(IChannelHandlerContext context)
 
     public async void SendMsg(DiscoveryMsg discoveryMsg)
     {
-        byte[] msgBytes;
-
+        IByteBuffer msgBuffer;
         try
         {
             if (_logger.IsTrace) _logger.Trace($"Sending message: {discoveryMsg}");
-            msgBytes = Serialize(discoveryMsg);
+            msgBuffer = Serialize(discoveryMsg, ByteBufferAllocator.PooledByteBufferAllocator);
```

Comment:
Recommend using `context.Allocator` instead in netty code. We don't change it right now but might be useful later.

Reply:
but the context is available here

Reply:
available or NOT available?

Reply:
sorry, NOT available.

Reply:
Can we pass it here?

Reply:
I meant can we pass it from `context` :)